Testscripts/Linux/SetupRDMA.sh

```diff
@@ -618,7 +629,7 @@ function Main() {
 		Verify_File $benchmark_bin
 	fi
 
-	if [ $benchmark_type == "OMB" && $$mpi_type != "mvapich" ]; then
+	if [[ $benchmark_type == "OMB" && $$mpi_type != "mvapich" ]]; then
```

Comment:
$$mpi_type - is this intentional?

Reply:
no. mistake. but harmless. Will fix it soon.

Reply:
Fixed in 5c9b1e184948b8eaa2fe2b77e38b942b2f0638df